### DISCUSSION TITLE
Is it possible to write config values to BMS?

### DISCUSSION THREAD

**Discussion by sunnysingh-84** [syssi/esphome-seplos-bms#89] [General]
Hi,

Wondering if its possible to write config values to Seplos BMS such as charge/discharge current limit? 

Would be handy if this could be done via Home Assistant instead of having to use BMS Studio.

**Comment by mager33** [comment]
... and switch the charge/discharge MOSFETS on/off would be verhy handy, too.

**Comment by midadami** [comment]
> Hi,
> 
> Wondering if its possible to write config values to Seplos BMS such as charge/discharge current limit?
> 
> Would be handy if this could be done via Home Assistant instead of having to use BMS Studio.

Its viable. The work just hasn't been done as it requires a lot of time and knowledge being done by someone who has access to the exact version of BMS.
Need to find what is usually sent from desktop app to replicate. Not sure if it is easier through BT or RS485.

**Comment by wyklikaniec83** [comment]
Any chances this could be looked at? I need to constantly manually adjust these values via BMS Studio in order to charge accordingly to the solar production (from separate/other inverter system)

**Comment by wyklikaniec83** [comment]
I can code in Python but not done any of this esphome development. If I get onto right track where to look, I can commit myself to look into it but will need some basics guidance where to start.
